components/Footer.js

```diff
@@ -14,22 +14,15 @@ export default function Footer() {
                             <li>Copy and paste the following command into the terminal, and press Enter:</li>
```

Comment:
In favour of: https://github.com/OpenAdaptAI/OpenAdaptWeb/pull/27/files

Lin 14 should be changed to:

`<li>Copy and paste the following command into the terminal, and press Enter (If Prompted for `User Account Control`, click 'Yes'):</li>`